src/App.tsx

```diff
@@ -1,12 +1,10 @@
 import { useState } from "react";
 import "./App.css";
-import initialData from "./assets/tasks-example.json";
+import initialTasks from "./assets/initial-tasks.json"
 import TableHeader from "./components/TableHeader";
 
 function App() {
-  const [tasks, setTasks] = useState(initialData);
-
-  console.log(tasks);
+  console.log(initialTasks);
```

Comment:
Sure

Reply:
Removed it, but then the code won't build since we'd have the unused variable intialTasks.
I get we don't want to have console logs in production. But there's no real use for the variable yet, and removing it would mean the ticket isn't resolved. So I'd suggest we leave it as is and then remove it on the next commit where someone actually uses that variable.

Reply:
sounds good!